rest/changestest/changes_api_test.go

```diff
@@ -3856,6 +3856,62 @@ func TestCacheCompactDuringChangesWait(t *testing.T) {
 	longpollWg.Wait()
 }
 
+func TestTombstoneResync(t *testing.T) {
```

Comment:
```suggestion
func TestResyncAllTombstonest *testing.T) {
```

Maybe more accurate?

Reply:
https://github.com/couchbase/sync_gateway/pull/6340/commits/b3111c121362d3ddd55aaa0acb9344f6c20178da